canonical-web-and-design/juju.is
# Build the homepage

Built the homepage from the [latest copy](https://docs.google.com/document/d/1vn6it-3px_KbplnajPlWPonU6W_xp4JdhE02kNQouZA/edit#).

## QA
- Open the demo
- Check the copy matches the copy doc

Fixes https://github.com/canonical-web-and-design/juju-squad/issues/1403
